cmd/coder/urls.go

```diff
@@ -14,13 +15,124 @@ import (
 )
 
 type urlsCmd struct{}
+type createURLCmd struct{}
+type delURLCmd struct{}
 
+// DevURL is the parsed json response record for a devURL from cemanager
 type DevURL struct {
+	ID     string `json:"id"`
 	URL    string `json:"url"`
 	Port   string `json:"port"`
 	Access string `json:"access"`
 }
 
+var urlAccessLevel = map[string]string{
+	"PRIVATE": "Only you can access",
+	"ORG":     "All members of your organization can access",
+	"AUTHED":  "Authenticated users can access",
+	"PUBLIC":  "Anyone on the internet can access this link",
+}
+
+func isAccessLevelValid(level string) bool {
+	_, ok := urlAccessLevel[level]
+	return ok
+}
+
+func (cmd createURLCmd) Spec() cli.CommandSpec {
+	return cli.CommandSpec{
+		Name:  "create",
+		Usage: "<env name> <port> [<access>]",
```

Comment:
Given that `access` is optional, I think it may make sense to have it as a flag. I think being able to optionally provide `--access=public` would be nice.

(See `syncCmd` for an example of how to register flags.)

Reply:
Sure, if we want that style I'll switch it up.

Reply:
I made it just --a, do you want it fully-spelled out ie., --access ?

Reply:
Single char arguments are usually single dashed, `-a` with a double dashed full word argument, `--access`